CHANGES

```diff
@@ -1,3 +1,30 @@
+SUMMARY OF RELEASE CHANGES FOR GREENBONE VULNERABILITY MANAGER
+==============================================================
+
+For detailed code changes, please visit
+ https://github.com/greenbone/gvm/commits/master
+or get the entire source code repository and view log history:
+$ git clone https://github.com/greenbone/gvm.git
+$ cd gvm && git log
+
+
+gvm 8.0+beta1 (2018-XX-XX)
+
+This is the first beta release of the gvm module 1.0 for the Greenbone
+Vulnerability Management (GVM) framework.
+
+It was renamed from openvas-manager to gvm (greenbone vulnerability manager).
```

Comment:
Can we use Greenbone Vulnerability Manager here for consistency?

Reply:
I also wondered about this, but I think he's matching the case of "gvm" (versus "GVM" above).